services/cinema_hall.py

```diff
@@ -1,5 +1,6 @@
 from django.db.models import QuerySet
 
+
```

Comment:
No need to have this line

Reply:
If I don't specify this, flake8 throws an error:
services\cinema_hall.py:4:27: F821 undefined name 'QuerySet'